apps/backend/.env.example

```diff
@@ -24,3 +24,12 @@ DB_POOL_MAX=10
 # ENCRYPT
 #
 SALT_ROUNDS=10
+
+#
+# AWS
+#
+ACCESS_KEY_ID=
+AMI_ID=
+REGION=
+SECRET_ACCESS_KEY=
+SECURITY_GROUP_ID=
```

Comment:
As we discussed, let's create security group using cloudformation, instead of passing the id